<?xml version="1.0"?>
<launch>
    <arg name="pkg"/>
    <arg name="pkg_full_path"/>
    <arg name="test_name" default="ts0_c0_r0_e0_0"/>
    
    <param name="/atf/package_name" value="$(arg pkg)"/>
    <param name="/atf/test_name" value="$(arg test_name)"/>
    <include file="$(arg pkg_full_path)/launch/application.launch"/>
    <node pkg="atf_core" type="sm_test.py" name="sm_test" args="standalone" required="true" output="screen"/>
    <node pkg="$(arg pkg)" type="application.py" name="application" args="standalone" required="true" output="screen"/>

</launch>
